docs/version0.12.md

```diff
@@ -36,6 +36,9 @@ The following new features and notable changes since v.0.11.0 are included in th
 - [RSA algorithm support for OpenSSL](#rsa-algorithm-support-for-openssl)
 - [`IBM_JAVA_OPTIONS` is deprecated](#ibm_java_options-is-deprecated)
 
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Warning:** Following the release of OpenJ9 0.12.0, an intermittent problem
+was identified with OpenSSL V1.1.x acceleration of the cryptographic Digest algorithm. For more information about the issue, which is under investigation, see [#4530](https://github.com/eclipse/openj9#4530). You can turn off the Digest algorithm by setting the [-Djdk.nativeDigest](djdknativedigest.md) system property to `false`.
```

Comment:
#4530 is not under investigation, it is closed. The #4530 link does not work.
Should this also mention a new release is available that disables nativeDigest by default?

Reply:
Updated, and link fixed.